domains/eryquicc.json

```diff
@@ -7,5 +7,16 @@
     },
     "record": {
         "CNAME": "eryquicc.github.io"
```

Comment:
You can't have any other records with `CNAME`
I'm assuming you wanted to replace `CNAME` with `A` records so that you could use `MX` and `TXT`
```suggestion
```